material-overrides/assets/stylesheets/moonbeam.css

```diff
@@ -1178,6 +1189,28 @@ html .md-typeset details.function div.tabbed-set,
 .md-feedback__icon:last-child:hover {
   background: rgba(226, 101, 96, 0.35);
 }
+/*divider between feedback and edit this button*/
+.divider {
+  width: 100%;
+  height: 1px;
+  background-color: var(--purple-dark-tint-20);
+  margin: 0.75em;
+}
+
+/*styles for the edit this page button*/
+.md-content__button {
+  margin: 0.75em auto 0 !important;
+  display: flex;
+  align-items: center;
+  gap: 0.5em;
+  font-size: 12px;
+  transition: color 0.3s ease;
+}
+
+.md-content__button.md-icon:hover {
+  color: var(--teal-light);
+  text-decoration: none;
```

Comment:
Moonbeam links typically use an opacity on hover.  So we should stick to that standard here. See the `--hover-opacity` variable